app/src/test/kotlin/compiler/semantic_analysis/VariablePropertiesAnalyzerTest.kt

```diff
@@ -440,4 +477,79 @@ class VariablePropertiesAnalyzerTest {
         assertAnalysisResults(input, expectedResults)
         assertDiagnostics(input, listOf())
     }
+
+    // czynność zewnętrzna() {
+    //     czynność wewnętrzna(x: Liczba = 123) { }
+    //     wewnętrzna(124)
+    // }
+
+    @Test
+    fun `test uncalled default parameter`() {
```

Comment:
nit
```suggestion
    fun `test unreferenced default parameter`() {
```
same for below